pkg/kyverno/test/test_command.go

```diff
@@ -205,7 +207,7 @@ results:
 	}
 	cmd.Flags().StringVarP(&fileName, "file-name", "f", "kyverno-test.yaml", "test filename")
 	cmd.Flags().StringVarP(&gitBranch, "git-branch", "b", "", "test github repository branch")
-
+	cmd.Flags().StringVarP(&testCase, "test-case-selector", "t", "", "run a specific test")
```

Comment:
We should document the format of the selector that we expect and how to specify policy/rule/resource etc.

Reply:
I made the usage more descriptively and also added an example of it. Let me know if you want any changes in it.